.github/workflows/build-test-e2e.yml

```diff
@@ -4,7 +4,7 @@ on: push
 jobs:
   build-and-run-e2e:
```

Comment:
I think you had a timeout at the application level, so maybe it's not necessary, but do you think one should be added at the job level?

Reply:
Oh good call! I'll add that now